src/image/image.js

```diff
@@ -420,15 +420,20 @@ p5.prototype.saveGif = function(pImg, filename) {
  *  as an argument to the callback function as an array of objects, with the
  *  size of array equal to the total number of frames.
  *
- *  Note that <a href="#/p5.Image/saveFrames">saveFrames()</a> will only save the first 15 frames of an animation.
+ *  Note that <a href="#/p5.Image/saveFrames">saveFrames()</a> will only save the first 15 seconds of an animation.
```

Comment:
I don't think we need this line anymore with the explanations below.

Reply:
@jesi-rgb Still have this pending then we can merge.

Reply:
Got it and also a minor typo!